packages/theming/apple-theme/src/appleTheme.ios.ts

```diff
@@ -126,6 +127,7 @@ export const BaseAppleLightThemeIOS: Theme = {
   typography: appleTypography(),
   spacing: appleSpacing(),
   components: appleComponents,
+  effects: defaultEffects(), // TODO: Saad, if this makes it into CR I'm counting on you to yell at me :D
```

Comment:
But what if I want it on the apple theme? :'(

Reply:
I haven't been paying attention to the theming stuff quite as much but why don't the apple and android themes base themselves off the default theme and then just override the apple/android portions as needed? You can still do applyTypography or appleSpacing calls to override it but it provides a fallback so that when we need a new section it doesn't require changes to all the platform specific themes.

This isn't a blocking comment for this PR, just curious why this approach was taken.

Reply:
Two reasons:

Short:
1) I couldn't wrap a ThemeReference in another ThemeReference. I could only wrap a theme or ThemeRecipe (I think that was the name). 
1) I didn't want edge cases where the apple theme didn't override some visuals from the default theme and you had some mixed tokens that came together unevenly. I'm not sure that's an actual problem but it was a thought. 

Long answer:

The Apple theme was initially implemented with a native module, so I needed a full theme reference with extra listeners and such. Rhe issue was I found that I couldn't set the base theme of a ThemeReference to another themeReference, only a theme. Not all the listeners transferred over, and I was getting bugs where my new theme would stay in light mode when it should be dark, etc. The Android Theme used the apple theme as a guide, so it was it's own ThemeReference too. I think the fact that a ThemeReference cannot wrap another ThemeReference was probably a bug, but I couldn't figure it out at the time and making a new ThemeReference seemed like the way to go. In the time interim, I managed to update `react-native-macos` with a new API to remove the need for a native module, so both the apple and android theme are defined entirely in JS, and could maybe be refactored as layers on the default theme.

Reply:
I want it in the apple theme, too Saad... :)  I just realized it would've probably been better organization to put the platform specific stuff in the platform specific places after I already shoved it all in the default theme.

I actually did in the default theme initially to be hackathon-quick because my mental model was similar to Jason's until I realized the platform themes don't fall back like thought, so I had to add it to the platform themes in a weird way.

Reply:
I'm pretty sure ThemeReferences can be layered, it's probably a lack of documentation on my part that made it seem they couldn't. For reference a ThemeReference has the following constructor:

```ts
constructor(base: TTheme | ThemeReference<TTheme, TPartial>, ...recipes: ThemeRecipe<TTheme, TPartial>[]) {
```

So if you want to do this in the future it looks like `const myTheme = new ThemeReference(baseTheme, ...myRecipes);`

They were designed to be layered as much as required. You could even go fluent -> apple -> ios / macos if you want. The second reason makes sense, though I wonder if some use of required types might help here. Ideally we want the compiler to catch those errors for us.

Reply:
I'll try having the apple theme be a layer/ThemeRecipe on the default theme again. I don't remember the exact issue I ran into, it might have also just been related to how we constructed the `testerTheme` in FluentTester

Reply:
Yea, the layering does work as far as i know--- see the examples I crafted in my AccChecker branch:  

https://github.com/FalseLobster/fluentui-react-native/commit/16913d5d97548adad399e1886fc82fb35d603f33